apps/core/src/hooks/useUnlockTimelockedObjectsTransaction.ts

```diff
@@ -5,16 +5,22 @@ import { useIotaClient } from '@iota/dapp-kit';
 import { createUnlockTimelockedObjectsTransaction } from '../utils';
 import { useQuery } from '@tanstack/react-query';
 import { useMaxTransactionSizeBytes } from './useMaxTransactionSizeBytes';
+import { useGetClockTimestamp } from './useGetClockTimestamp';
 
 export function useUnlockTimelockedObjectsTransaction(address: string, objectIds: string[]) {
     const client = useIotaClient();
     const { data: maxSizeBytes = Infinity } = useMaxTransactionSizeBytes();
+    const { data: clockTimestampMs } = useGetClockTimestamp();
 
     return useQuery({
         // eslint-disable-next-line @tanstack/query/exhaustive-deps
         queryKey: ['unlock-timelocked-objects', address, objectIds],
         queryFn: async () => {
-            const transaction = createUnlockTimelockedObjectsTransaction({ address, objectIds });
+            const transaction = createUnlockTimelockedObjectsTransaction({
+                address,
+                objectIds,
+                isClockTimestampEnabled: !!clockTimestampMs,
```

Comment:
Why would it not be available?

Reply:
The clock package is a system package, so it should always exist, the whole `isClockTimestampEnabled` seems unnecessary

Reply:
What was not available in Testnet was the move call `unlock_with_clock`, not the clock itself, the clock is always there in every network, see https://explorer.rebased.iota.org/object/0x0000000000000000000000000000000000000000000000000000000000000006 and https://github.com/iotaledger/iota/blob/145ac5977a6d2e4e1ad99fb024881824e0c9f314/crates/iota-framework/packages/iota-framework/sources/clock.move#L6